tilecloud_chain/USAGE.rst

```diff
@@ -283,8 +283,8 @@ The process is a set of names processes, and each one has a list of commands dec
     process:  # root process config
         optipng:  # the process command
         -   cmd: optipng %(args)s -q -zc9 -zm8 -zs3 -f5 -o %(out)s %(in)s  # the command line
-            need_out: true  # if false the command rewrite the input file, default to false
-            arg:  # argument used with the different log switches, all default to ''
+            need_out: true  # if false the command rewrite the input file, default is false
+            arg:  # argument used with the different log switches, all default is ''
```

Comment:
does "all" refer to "all arguments"? In this case it is plural.
So:
"all arguments default to"
"for each argument, default is"

Reply:
Unclear => `in all cases default is ''`